README.md

```diff
@@ -50,55 +50,49 @@ To use the latest build (pre-release), don't forget to enable the download of sn
 
 ### Usage Example
 We provide database drivers that intercept calls to real database drivers and swap out secret IDs for actual login credentials.
```

Comment:
We provide database drivers that intercept calls to real database drivers and replace secret IDs with actual login credentials.